msg/SemanticClass.msg

```diff
@@ -0,0 +1,10 @@
+# A key value pair that maps an integer class_id to a string class label.
+# The class_id should be interpreted as the pixel value corresponding to a 
+# given class name in a segmentation mask
+
+# Integer value corresponding to the value of pixels belonging
+# to a given class in a segmentation mask
+uint16 class_id
```

Comment:
Why is this a uint16 when the `data` field in `SemanticSegmentation.msg` is only uint8?

Limiting the number of classes to 255 is probably bad, but doubling the message size for the segmentation image by making it a uint16 is equally bad. Anyhow, the two should be consistent.

Reply:
On second thought, it's probably better to leave it as uint16. If we use a `sensor_msgs/Image` as the segmentation image format, we can choose between mono8 and mono16 encodings as appropriate there.

Reply:
Can't speak to that exactly without his comment, but there can easily be more than 255 class values in segmentation algorithms.

Reply:
My bad, they must be consistent. I will use `uint16` for the final version of the PR